edenai_apis/apis/deepgram/deepgram_api.py

```
@@ -262,3 +266,135 @@ async def audio__atext_to_speech(
+
+        if provider_params.get("sample_rate"):
+            base_url += f"&sample_rate={provider_params['sample_rate']}"
+
+        headers = {
+            "Authorization": f"Token {self.api_key}",
+            "Content-Type": "application/json",
+        }
+
+        payload = {"text": text}
+
+        response = requests.post(base_url, headers=headers, json=payload)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)